### PR TITLE
Docs of CamelCase

### PR DESCRIPTION
**Pull request by dereuromark**
Refs https://api.cakephp.org/4.0/class-Cake.Utility.Inflector.html#camelize

We shouldnt use a word, that has no good meaning and is not in line with the current API and how people use the casings.

Most people don't know what Pascal case means or is supposed to be. CamelCase and camelBacked is an agnostic and visual way to describe it and thus a good default anyway.
It makes sense to not confuse people already used to those very good default style names.
https://en.wikipedia.org/wiki/Camel_case lists PascalCase as well, but as I said: It has less visual and thus less useful meaning.

### CODE REVIEW

File: en/intro/conventions.rst [hunk 1]
```diff
@@ -182,14 +182,14 @@ by creating classes and files that you'd need to create anyway.
...
 | Entity     | Articles.php                | ArticlesCategories.php           | Entity class names are singular,                     |
-|            |                             |                                  | PascalCased: Article and ArticleCategory             |
+|            |                             |                                  | CamelCased: Article and ArticleCategory             |
 +------------+-----------------------------+----------------------------------+------------------------------------------------------+
 | Class      | ArticlesController          | ArticlesCategoriesController     |                                                      |
 +------------+-----------------------------+----------------------------------+------------------------------------------------------+
-| Controller | ArticlesController          | ArticlesCategoriesController     | Plural, PascalCased, end in Controller               |
+| Controller | ArticlesController          | ArticlesCategoriesController     | Plural, CamelCased, end in Controller               |
```
Comment (markstory) [inline en/intro/conventions.rst:192]:
```suggestion
| Controller | ArticlesController          | ArticlesCategoriesController     | Plural, CamelCased, end in Controller                |
```

File: en/intro/conventions.rst [hunk 2]
```diff
@@ -182,14 +182,14 @@ by creating classes and files that you'd need to create anyway.
...
 +------------+-----------------------------+----------------------------------+------------------------------------------------------+
 | Table      | ArticlesTable.php           | ArticlesCategoriesTable.php      | Table class names are plural,                        |
-|            |                             |                                  | PascalCased and end in Table                         |
+|            |                             |                                  | CamelCased and end in Table                         |
 +------------+-----------------------------+----------------------------------+------------------------------------------------------+
 | Entity     | Articles.php                | ArticlesCategories.php           | Entity class names are singular,                     |
-|            |                             |                                  | PascalCased: Article and ArticleCategory             |
+|            |                             |                                  | CamelCased: Article and ArticleCategory             |
```
Comment (markstory) [inline en/intro/conventions.rst:188]:
```suggestion
|            |                             |                                  | CamelCased: Article and ArticleCategory              |
```

File: en/intro/conventions.rst [hunk 3]
```diff
@@ -182,14 +182,14 @@ by creating classes and files that you'd need to create anyway.
 | File       | ArticlesController.php      | ArticlesCategoriesController.php |                                                      |
 +------------+-----------------------------+----------------------------------+------------------------------------------------------+
 | Table      | ArticlesTable.php           | ArticlesCategoriesTable.php      | Table class names are plural,                        |
-|            |                             |                                  | PascalCased and end in Table                         |
+|            |                             |                                  | CamelCased and end in Table                         |
```
Comment (markstory) [inline en/intro/conventions.rst:185]:
```suggestion
|            |                             |                                  | CamelCased and end in Table                          |
```

File: en/development/routing.rst [hunk 4]
```diff
@@ -685,7 +685,7 @@ The current prefix will be available from the controller methods through
 ``$this->request->getParam('prefix')``
 
 When using prefix routes it's important to set the ``prefix`` option, and to
-use the same camel cased format that is used in the ``prefix()`` method. Here's
+use the same camel-cased format that is used in the ``prefix()`` method. Here's
```
Comment (othercorey) [inline en/development/routing.rst:688]:
```suggestion
use the sameCamelCased format that is used in the ``prefix()`` method. Here's
```

Should we be consistent everywhere? If we're going to commit to the style, let's do it, right?

File: en/development/routing.rst [hunk 5]
```diff
@@ -729,7 +729,7 @@ This would link to a controller with the namespace ``App\\Controller\\Admin\\MyP
 
 .. note::
 
-    The prefix is always camel cased here, even if the routing result is dashed.
+    The prefix is always camel-cased here, even if the routing result is dashed.
```
Comment (othercorey) [inline en/development/routing.rst:732]:
```suggestion
    The prefix is always CamelCased here, even if the routing result is dashed.
```

File: en/development/routing.rst (hunk 4 above)
Comment (dereuromark) [reply en/development/routing.rst:688]: There is a Whitespace issue

**Pull request merged**